digital_land/datatype/wkt.py

```diff
@@ -209,24 +214,27 @@ def normalise(self, value, default="", issues=None):
             # fixed/normalised geometry. To reduce precision,
             # round trip the geometry through shapely with 6 dp precision.
 
-            _wkt = dump_wkt(geometry, precision=6)
+            _wkt, issue = dump_wkt(geometry, precision=6)
+            if issue:
+                issues.log("geometry is too precise - fixed", "")
+
             geometry = shapely.wkt.loads(_wkt)
             validity = False
             i = 0
             while i < 3 and not validity:
                 geometry, issue = normalise_geometry(geometry)
                 if issue:
-                    issues.log("invalid geometry", issue)
+                    issues.log("invalid geometry provided - fixed", issue)
 
                 if not geometry:
                     return default
 
-                _wkt = dump_wkt(geometry)
+                _wkt, issue = dump_wkt(geometry)
                 geometry = shapely.wkt.loads(_wkt)
                 validity = geometry.is_valid
                 i += 1
 
         if not geometry:
             return default
 
-        return dump_wkt(geometry)
+        return dump_wkt(geometry)[0]
```

Comment:
we could update this area. Right now as you can see if the geometry is still invalid we still actually include it. I would suggest a simple .is_valid check to see if it is finally valid and then to remove it if it's not and record an issue.

Reply:
Done